app/uk/gov/hmrc/alcoholdutyaccount/controllers/actions/AuthorisedAction.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.alcoholdutyaccount.controllers.actions
+
+import com.google.inject.Inject
+import play.api.http.Status.UNAUTHORIZED
+import play.api.libs.json.Json
+import play.api.mvc.Results.Unauthorized
+import play.api.mvc._
+import uk.gov.hmrc.auth.core.AffinityGroup.Organisation
+import uk.gov.hmrc.auth.core.AuthProvider.GovernmentGateway
+import uk.gov.hmrc.auth.core.CredentialStrength.strong
+import uk.gov.hmrc.auth.core._
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendHeaderCarrierProvider
+import uk.gov.hmrc.play.bootstrap.backend.http.ErrorResponse
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait AuthorisedAction
+    extends ActionBuilder[Request, AnyContent]
+    with BackendHeaderCarrierProvider
+    with ActionFunction[Request, Request]
+
+class BaseAuthorisedAction @Inject() (
+  override val authConnector: AuthConnector,
+  val parser: BodyParsers.Default
+)(implicit val executionContext: ExecutionContext)
+    extends AuthorisedAction
+    with BackendHeaderCarrierProvider
+    with AuthorisedFunctions {
+
+  private val alcoholDutyEnrolment = "HMRC-AD-ORG"
```

Comment:
Add this to config as we do in the frontend please: https://github.com/hmrc/alcohol-duty-returns-frontend/blob/main/conf/application.conf#L107